CalibMuon/DTCalibration/src/DTCalibMuonSelection.cc

```diff
@@ -53,7 +53,7 @@ bool DTCalibMuonSelection::filter(edm::Event& iEvent, const edm::EventSetup& iSe
 }
 
 // ------------ method called once each job just before starting event loop  ------------
-void DTCalibMuonSelection::beginJob() {}
+void DTCalibMuonSelection::beginStream(edm::StreamID) {}
```

Comment:
Same thing as in my comment above

Reply:
as above, this is not harming anything (could indeed be deleted completely).

Reply:
Thanks Marco!